src/protocol/logs.rs

```diff
@@ -0,0 +1,212 @@
+pub mod events {
+    use serde::{Deserialize, Serialize};
+    use crate::protocol::runtime::methods::{StackTrace, RemoteObject};
+    use crate::protocol::types::{JsInt};
+
+    #[derive(Deserialize, Serialize, Debug, Clone, PartialEq, Eq)]
+    #[serde(rename_all = "lowercase")]
+    pub enum  LogEntrySource {
+        XML,
```

Comment:
I'd prefer if these were in `UpperCamelCase` to match the style of enum variants in the rest of the project and also [these guidelines](https://rust-lang-nursery.github.io/api-guidelines/naming.html).

Reply:
heh, even though I happened to comment on the `XML` line which is one of the few that wouldn't need to change ;)

Reply:
Sure, I'll refactor the Enums, thanks for the link to guidelines.

Reply:
Refactored to `UpperCamelCase`, also added forgotten `pub` in one of the enums. Also ran formatting with `cargo fmt`